apps/web/src/ee/clerk/providers/ClerkProvider.tsx

```diff
@@ -27,6 +27,8 @@ const localization = {
   },
 };
 
+const ALLOWED_REDIRECT_ORIGINS = ['http://localhost:2022', new URL(location.href).origin];
```

Comment:
Can you please try this and see if it works?

```suggestion
const ALLOWED_REDIRECT_ORIGINS = ['http://localhost:*'];
```

Reply:
great suggestion, applied